### PR TITLE
Feature/verse of the day

### PR DESCRIPTION
**Pull request by rorJeremy**

### CODE REVIEW

File: bibleApp/src/components/VerseList.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import VerseCard from  './VerseCard';
+
+
+class VerseList extends Component {
+	state = { 
+		verseOfTheDay: {},
+	};
+	// componentWillMount method automatically gets executed as soon as this component is about to get rendered to the screen.
+	componentWillMount() {
+		axios.get('https://www.ourmanna.com/verses/api/get/?format=json')
```
Comment (mlohstroh) [inline bibleApp/src/components/VerseList.js]: What's wrong with using the fetch function built into react native? 

https://facebook.github.io/react-native/docs/network.html

Comment (mlohstroh) [inline bibleApp/src/components/VerseList.js]: To clarify, I'd rather use built in functionality than import another library.

Comment (rorJeremy) [inline bibleApp/src/components/VerseList.js]: I just like axios because it reminds me of httparty, and has good documentation. Have you looked at it?

**Comment by rorJeremy** [comment]
Am I merge worthy now?

**Comment by mlohstroh** [comment]
We will fix the tabs/spaces issues later. Probably best in person.

**Pull request merged**